src/extensions/default/JavaScriptCodeHints/HintUtils.js

```diff
@@ -145,7 +145,7 @@ define(function (require, exports, module) {
             t.kind = kind;
             t.origin = "ecmascript";
             if (kind === "string") {
-                if (/[\\\\]*[^\\]"/.test(t.value)) {
+                if (/[^\\]"/.test(t.value)) {
```

Comment:
This looks bit scary to me, but didn't manage to find any differences after running some tests cases by hand 🤔